DOCS/man/af.rst

```diff
@@ -193,6 +193,17 @@ Available filters are:
         If you want to go up or down by semi-tones, use 1.059463094352953 and
         0.9438743126816935
 
+``chromium``
```

Comment:
Do we actually want to call it "chromium"? what if chromium changes its algorithm tomorrow?

Reply:
That's a good point. I'm fine with anything. Maybe `scaletempo2` because it is a very similar algorithm to that one?

Reply:
> Maybe `scaletempo2`

Funnily enough, I thought of exactly this name too, though I wasn't sure if I should mention it as it's a bit weird :)

Reply:
i think better stay with chromium because it very well made

Reply:
I would call it `wsola`, because that seems to be what the algorithm itself is called in the literature. If need be you can mention that chromium uses it in the documentation for that option.

Reply:
No it's confusing there's a lot WSOLA algorithms that sounds worse than chromium

Reply:
Oh, I see. Well then, I guess it's fine. I just think it's a weird name for an audio algorithm, after all. "chromium" could mean any number of things. It doesn't really communicate what this exact filter is doing at all.

I'd be fine with the naming if it was in the context of an option like ``--audio-pitch-shift-algorithm=chromium`, but as it stands it's basically a stand-alone audio filter, `--af=chromium`. Maybe it could be `wsola-chromium` to distinguish this particular WSOLA variant from others?

Reply:
`wsola-chromium` sounds good

Reply:
`scaletempo-ng`?

Reply:
It's subjective, but in retrospect, i like `scaletempo2` the most, because It's an internal filter which is an alternative to scaletempo and has the same goal.

Maybe off topic, doesn't ffmpeg have such things with its filters?

Reply:
@avih Ok, then I will change it to `scaletempo2`.
ffmpeg also has this, but apparently it can't be used due to a missing API (https://github.com/mpv-player/mpv/issues/4418#issuecomment-636095336).

Reply:
Does original scaltempo implementation still have any useful use case now? How about replacing existing scaletempo and keeping the same name?

Reply:
I use scaletempo instead of rubberband currently. Perhaps in the event that this patch is merged and works as advertised, then perhaps a filter rename could be a thing.